src/components/layout/_Header.svelte

```diff
@@ -1,13 +1,18 @@
 <script lang="ts">
```

Comment:
Wo besteht hier der Unterschied zum Header in der Landing page? Wenn das nur wegen dem Settings Button sein sollte, dann könnte man das auch dynamisch in einer Komponente machen und dann die Settings nur anzeigen, wenn der Nutzende angemeldet ist

Reply:
Ich denk mal, dass die zwei Header sich mit der Zeit etwas verändern werden, weswegen ich schon vorab zwei verschiedene gemacht habe. Würde das gerne so beibehalten